clang/include/clang/Format/Format.h

```diff
@@ -158,10 +158,20 @@ struct FormatStyle {
     /// \code
     ///   #define A \
     ///     int aaaa; \
+    ///   \
     ///     int b; \
     ///     int dddddddddd;
     /// \endcode
     ENAS_DontAlign,
+    /// Don't align escaped newlines.
```

Comment:
the difference is the one empty line
`    \`
vs
`\`

Reply:
Yep, I see the different in the behavior. I mean that the text "Don't align escaped newlines" is identical, maybe we can change it so that it represents the other behavior?